blackjack.py

```diff
@@ -90,14 +90,13 @@ def print_entire_hand(self):
 
     def check_natural_win(self):
         """
-        Checks if a hand has two cards
-        First checks if hand has two cards
-        :return: True if Ace & face card, False otherwise
-        """
-        if len(self.hand) == 2:
-            return ((self.hand[0].value == 'Ace' and self.hand[1].value in ('Jack', 'King', 'Queen')) or
-                    (self.hand[1].value == 'Ace' and self.hand[0].value in ('Jack', 'King', 'Queen')))
-        return False
+        Makes a set of all values of cards in hand
+        Checks length of set is 2, Ace is in hand and set is subset of facecards
+        :return: True if set has length of two & another face card else False
+        """
+        hand_set = set(card.value for card in self.hand)
+        return len(hand_set) == 2 and 'Ace' in hand_set \
```

Comment:
You want to avoid backlashes, they could get you into trouble. Also, a convention is to break chained conditions like that after the keywords (i.e. you'd end a line with `and` or `or` and not start it).
Before explaining why, here's what it should look like:
```
        return (len(hand_set) == 2 and 'Ace' in hand_set and
                hand_set < {'Ace', 'Jack', 'King', 'Queen'})
```

So why the hate towards backslashes?
From [PEP8](https://www.python.org/dev/peps/pep-0008/):

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Here's one of the dangers of backslashes:
```
In [1]: "a" + \
   ...:     "b"
Out[1]: 'ab'

In [2]: "a" + \ 
   ...:     "b"
  File "<ipython-input-2-d7bada184e9a>", line 1
    "a" + \
            ^
SyntaxError: unexpected character after line continuation character
```

What happened? A space was added after the backslash, which obviously is hard to see, and now everything is broken.

The [Python documentation](https://docs.python.org/2/howto/doanddont.html#using-backslash-to-continue-statements) also warns about this behavior.